src/browser/BrowserService.cpp

```diff
@@ -50,6 +50,9 @@ static int KEEPASSXCBROWSER_DEFAULT_ICON = 1;
 const char BrowserService::LEGACY_ASSOCIATE_KEY_PREFIX[] = "Public Key: ";
 static const char KEEPASSHTTP_NAME[] = "KeePassHttp Settings";
 static const char KEEPASSHTTP_GROUP_NAME[] = "KeePassHttp Passwords";
+// Extra entry related options saved in custom data
+const char BrowserService::OPTION_SKIP_AUTO_SUBMIT[] = "BrowserSkipAutoSubmit";
```

Comment:
For these you can also use a QStringLiteral which builds a QString object at compile time. This is my preferred method over cstrings

Reply:
Only reason that was not used is that QLatin1String doesn't handle QStrings directly. If we must prefer QStringLiteral, we must make an additional wrapper function for QLatin1String.

Reply:
Using `.toUtf8()` for `QStringLiteral` was enough.